src/GFExcelAdmin.php

```diff
@@ -17,11 +20,11 @@
 
 class GFExcelAdmin extends GFAddOn
 {
-    const BULK_DOWNLOAD = 'gfexcel_download';
+    public const BULK_DOWNLOAD = 'gfexcel_download';
 
-    private static $_instance = null;
+    private static $_instance;
 
-    protected $_min_gravityforms_version = "2.0";
+    protected $_min_gravityforms_version = '2.0';
 
     protected $_capabilities_form_settings = ['gravityforms_export_entries'];
```

Comment:
I think this needs to be a string? At least starting with 2.5.

Reply:
This also works. So changed it. Also added this these rights for the general settings page.